platform/migration/platform-migration-commands/src/main/java/org/codice/ddf/migration/commands/ExportCommand.java

```diff
@@ -25,8 +27,11 @@
  * are exported to the directory specified in the implementation of the service
  * (see {@link ConfigurationFileDirectory} for an example).
  */
-@Command(scope = PlatformCommands.NAMESPACE, name = "config-export", description = "Exports configurations")
-public class ExportCommand extends PlatformCommands {
+@Command(scope = MigrationCommands.NAMESPACE, name = "export", description = "Exports the entire system in order "
+    + "to prepare for a migration to a new system. Exported items may include configurations, files, catalog contents, etc.. "
```

Comment:
Do we need the two `.` after `etc`? Looks weird to me but I'm no a grammar expert so it's up to you. Just wanted to make sure it wasn't just a typo.